trie/triedb/pathdb/nodebufferlist.go

```diff
@@ -0,0 +1,864 @@
+package pathdb
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/VictoriaMetrics/fastcache"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/trie/trienode"
+	"github.com/ethereum/go-ethereum/trie/triestate"
+)
+
+const (
+	// mergeMultiDifflayerInterval defines the interval to collect nodes to flush disk.
+	mergeMultiDifflayerInterval = 3
+
+	// DefaultProposeBlockInterval defines the interval of op-proposer proposes block.
+	DefaultProposeBlockInterval = 3600
+
+	// DefaultReserveMultiDifflayerNumber defines the default reserve number of multiDifflayer in nodebufferlist.
+	DefaultReserveMultiDifflayerNumber = 3
```

Comment:
Is this value configurable through command line arguments? Perhaps we need to customize it.